src/file/file.grpc.ts

```diff
@@ -40,13 +47,22 @@ export class FileMethods {
    * UpdateFiles updates a list of files and responses with a list of the files' id
    * which succeeded the operation.
    */
-  public static async UpdateFiles(call: ServerUnaryCall<{ partialFile: Partial<IFile>, idList: string[] }>) {
+  public static async UpdateFiles(call: ServerUnaryCall<{ partialFile: Partial<IFile>, idList: string[] }>)
+  : Promise<{ updated: string[], failedFiles: FailedFile[] }> {
     const { updated, failed } = await FileService.updateMany(call.request.idList, call.request.partialFile);
     const traceID = getCurrTraceId();
+    const failedFiles: FailedFile[] = [];
     for (let i = 0; i < failed.length; i++) {
+      let name: string = '';
+      try {
+        const file = await FileService.getById(failed[i].id);
+        name = file.name;
+      } catch (e) {}
```

Comment:
Empty catch?

Reply:
Yes. since it already handles an error, there is no use for handling another error. 
On the other hand, removing the 'try, catch' could compromise the service.

Reply:
remove all name related lines.